tutorials/index.ipynb

```diff
@@ -0,0 +1,104 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting Started with [pyfod](https://github.com/prmiles/pyfod/wiki)\n",
+    "Author(s): Paul R. Miles, Graham T. Pash | April 8, 2019\n",
+    "# Introduction\n",
+    "The [pyfod](https://github.com/prmiles/pyfod/wiki) package is a Python program for numerically approximating fractional order derivatives (fod).  The definitions available within the package include:\n",
+    "\n",
+    "- Riemann-Liouville\n",
+    "$$\n",
+    "D_{RL}^\\alpha[f(t)] = \\frac{1}{\\Gamma(n-\\alpha)}\\frac{d^n}{dt^n}\\int_{t_0}^{t_f}\\frac{f(s)}{(t_f-s)^\\alpha}ds \\;\\; \\text{where} \\;\\; n=\\lceil\\alpha\\rceil.\n",
+    "$$\n",
+    "- Caputo\n",
+    "$$\n",
+    "D_C^\\alpha[f(t)] = \\frac{1}{\\Gamma(n-\\alpha)}\\int_{t_0}^{t_f}\\frac{f^{(n)}(s)}{(t_f-s)^\\alpha}ds \\;\\; \\text{where} \\;\\; n=\\lceil\\alpha\\rceil.\n",
+    "$$\n",
+    "- Grünwald-Letnikov\n",
+    "$$\n",
+    "D_G^\\alpha [f(t)]=\\lim_{h\\rightarrow 0} \\frac{1}{h^\\alpha}\\sum_{0\\leq m< \\infty}(-1)^m{\\scriptsize\\left(\\begin{array}{c}\\alpha\\\\m\\end{array}\\right)}f(t-mh).\n",
+    "$$\n",
+    "\n",
+    "**Note:** At this time, the Riemann-Liouville definition is limited to cases where $\\alpha \\in [0, 1)$.\n",
+    "\n",
+    "# Installation\n",
+    "This code can be found on the [Github project page](https://github.com/prmiles/pyfod).  To install the master branch directly from Github,\n",
+    "\n",
+    "```\n",
+    "    pip install git+https://github.com/prmiles/pyfod.git\n",
+    "```\n",
+    "**Note:** Current examples generated using development branch of repository.\n",
+    "\n",
+    "\n",
+    "# Examples of Evaluating Fractional Derivatives\n",
+    "The following general examples demonstrate how to evaluate the following fractional derivative definitions:\n",
+    "- [Riemann-Liouville](https://nbviewer.jupyter.org/github/prmiles/notebooks/blob/master/pyfod/riemann_liouville.ipynb)\n",
+    "- Caputo (Pending)\n",
+    "- Grünwald-Letnikov (Pending)\n",
+    "\n",
+    "# Examples of Quadrature Methods\n",
+    "- [Quadrature Demos](https://nbviewer.jupyter.org/github/gtpash/notebooks/blob/master/pyfod/quadrature_demos.ipynb)"
```

Comment:
This link needs to be updated.
https://nbviewer.jupyter.org/github/prmiles/pyfod/blob/master/tutorials/quadrature_demos.ipynb

Reply:
Updated.